src/gui_w32.c

```diff
@@ -7521,15 +7521,16 @@ lpwAlign(
     static int
 nCopyAnsiToWideChar(
     LPWORD lpWCStr,
-    LPSTR lpAnsiIn)
+    LPSTR lpAnsiIn,
+    BOOL enc)
```

Comment:
I think explanation for this parameter is needed. How about adding a comment like this?
`/* If TRUE, 'encoding' is used for lpAnsiIn. If FALSE, current ACP is used for lpAnsiIn. */`